sample/HttpTrigger-Powershell/run.ps1

```diff
@@ -1,10 +1,10 @@
-if ($Env:req_query_name) 
```

Comment:
Will you also be handling the updates in the templates repo? There are several PS templates I believe. https://github.com/Azure/azure-webjobs-sdk-templates/blob/dev/Templates/QueueTrigger-Powershell/run.ps1